modules/3d-tiles/src/lib/parsers/helpers/parse-3d-implicit-tiles.ts

```diff
@@ -10,6 +13,32 @@ const SUBDIVISION_COUNT_MAP = {
   OCTREE: OCTREE_DEVISION_COUNT
 };
 
+function getCTile(curTile, midZ, sizeZ, index) {
+  let cTile;
+  if (curTile && curTile.boundingVolume && curTile.boundingVolume.box) {
+    if (typeof curTile.boundingVolume.cellId === 'undefined')
+      curTile.boundingVolume.cellId = getIdFromToken(curTile.boundingVolume.s2bv.token);
+    // Get box from childCellId
+    const childCellId = getS2ChildCellId(curTile.boundingVolume.cellId, index);
+    const childToken = getTokenFromId(childCellId);
+
+    // Clone object
+    const s2bv: S2BoundingVolume = JSON.parse(JSON.stringify(curTile.boundingVolume.s2bv));
```

Comment:
The parse - stringify is not optimal approach. I suggest make a copy function. S2BoundingVolume object is not so long to do that.